packages/app/auth/hooks/useUser.ts

```diff
@@ -0,0 +1,14 @@
+import { queryTrpc } from 'app/trpc';
+import { useStorage } from 'app/hooks/storage/useStorage';
+
+export const useUserQuery = () => {
+  const [[isLoading, user]] = useStorage('session');
```

Comment:
this data comes back differently from secure storage versus local storage on web. we might want to make this a bit easier to follow too